grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -106,6 +106,11 @@
             pendingTaskUpdater = AtomicReferenceFieldUpdater.newUpdater(
             ArmeriaClientCall.class, Runnable.class, "pendingTask");
 
+    @SuppressWarnings("rawtypes")
+    private static final AtomicReferenceFieldUpdater<ArmeriaClientCall, Boolean>
+            closedUpdater = AtomicReferenceFieldUpdater.newUpdater(
```

Comment:
We usually use `AtomicIntegerFieldUpdater` with `0` and `1` for the boolean field.
```
private volatile int closed;

if (closed()) {...}

private boolean closed() {
    return closed == 1;
}
```